src/stores/linkStore.ts

```diff
@@ -35,10 +34,11 @@ export class LinkStore {
     @action
     public setLink = (link: ILink) => {
         this._link = link;
-        this._undoObjects = [{
+        const undoObject: UndoObject = {
```

Comment:
Would it not be more natural to use the words linkGeometryState: LinkGeometryState. UndoObject feels a bit abstract.

Then we could have syntax such as:
GeometryUndoStack.addItemToUndoStack(linkGeometryState);
instead of 
UndoStore.addUndoObject(undoObject).

Reply:
Lets go with: this._undoStore.addItem(currentState);